config.xml

```diff
@@ -119,17 +119,27 @@
         <splash height="1792" src="resources/ios/splash/Default-1792h~iphone.png" width="828" />
         <splash height="828" src="resources/ios/splash/Default-Landscape-1792h~iphone.png" width="1792" />
     </platform>
-    <plugin name="cordova-plugin-statusbar" spec="2.4.2" />
-    <plugin name="cordova-plugin-device" spec="2.0.2" />
-    <plugin name="cordova-plugin-splashscreen" spec="5.0.2" />
-    <plugin name="cordova-plugin-ionic-webview" spec="^5.0.0" />
-    <plugin name="cordova-plugin-ionic-keyboard" spec="^2.0.5" />
-    <plugin name="cordova-plugin-camera" spec="^4.0.3" />
-    <plugin name="cordova-plugin-geolocation" spec="^4.0.2">
+    <engine name="ios" />
+    <plugin name="cordova-plugin-statusbar" spec="^2.4.2" />
+    <plugin name="cordova-plugin-device" spec="^2.0.2" />
+    <plugin name="cordova-plugin-splashscreen" spec="^5.0.2" />
+    <plugin name="cordova-plugin-ionic-webview" spec="^5.0.1" />
+    <plugin name="cordova-plugin-ionic-keyboard" spec="^2.2.0" />
+    <plugin name="cordova-plugin-camera" spec="^4.1.0">
+        <variable name="ANDROID_SUPPORT_V4_VERSION" value="27.+" />
+    </plugin>
+    <plugin name="cordova-plugin-geolocation" spec="^4.1.0">
         <variable name="GEOLOCATION_USAGE_DESCRIPTION" value="To set observation locations" />
+        <variable name="GPS_REQUIRED" value="true" />
+    </plugin>
+    <plugin name="cordova-plugin-advanced-http" spec="^3.3.1">
+        <variable name="ANDROIDBLACKLISTSECURESOCKETPROTOCOLS" value="SSLv3,TLSv1" />
     </plugin>
-    <plugin name="cordova-plugin-advanced-http" spec="^3.1.0" />
     <plugin name="cordova-android-support-gradle-release" spec="^3.0.1">
         <variable name="ANDROID_SUPPORT_VERSION" value="27.0.0" />
     </plugin>
+    <plugin name="cordova-sqlite-storage" spec="^6.1.0" />
+    <plugin name="cordova-plugin-googlemaps" spec="^2.7.1" />
```

Comment:
I think this plugin broke the iOS build - Might need to check if this builds on XCode first